src/xdotoolify.js

```diff
@@ -326,6 +326,36 @@ _Xdotoolify.prototype.autoType = function(selector, text, relpos, timeout) {
   }
   return this;
 };
+
+_Xdotoolify.prototype.sleepUntil = function(predicate, timeout) {
+  this.operations.push({
+    type: 'sleepUntil',
+    predicate: predicate,
+    timeout: timeout || 15000
```

Comment:
I think the default timeout should be lower, I think in most real scenarios things don't take more than 3 secs

Reply:
Fixed